test/TestUtils.ts

```diff
@@ -6,9 +6,26 @@ import { RelayHubInstance, StakeManagerInstance } from '../types/truffle-contrac
 import HttpWrapper from '../src/relayclient/HttpWrapper'
 import HttpClient from '../src/relayclient/HttpClient'
 import { configureGSN } from '../src/relayclient/GSNConfigurator'
+import { ether } from '@openzeppelin/test-helpers'
+import fs from 'fs'
 
 const localhostOne = 'http://localhost:8090'
 
+// from: https://stackoverflow.com/questions/18052762/remove-directory-which-is-not-empty
+export function cleanFolder (folder: string): void {
```

Comment:
You can simply do `fs.rmdirSync(folder, {recursive: true})`

Reply:
fixed.